sklearn/manifold/t_sne.py

```diff
@@ -645,10 +645,10 @@ class TSNE(BaseEstimator):
     """
 
     def __init__(self, n_components=2, perplexity=30.0,
-                 early_exaggeration=4.0, learning_rate=1000.0, n_iter=1000,
+                 early_exaggeration=4.0, learning_rate=200.0, n_iter=2000,
```

Comment:
Maybe not best to do in this PR, but these lower learning rates may be a better default choice (https://www.reddit.com/r/MachineLearning/comments/47kf7w/scikitlearn_tsne_implementation/)